eva/optimizer/optimizer_utils.py

```diff
@@ -76,12 +73,19 @@ def metadata_definition_to_udf_metadata(metadata_list: List[Tuple[str, str]]):
 
 def extract_equi_join_keys(
     join_predicate: AbstractExpression,
-    left_table_aliases: List[str],
-    right_table_aliases: List[str],
+    left_table_aliases: List[Alias],
+    right_table_aliases: List[Alias],
 ) -> Tuple[List[AbstractExpression], List[AbstractExpression]]:
     pred_list = to_conjunction_list(join_predicate)
     left_join_keys = []
     right_join_keys = []
+    left_table_alias_strs = []
```

Comment:
We can use list comprehension to make it one-liner and pythonic

Reply:
Done